editor/js/libs/app.js

```diff
@@ -146,7 +161,23 @@ var APP = {
 
 			for ( var i = 0, l = array.length; i < l; i ++ ) {
 
-				array[ i ]( event );
+				try {
+
+					array[ i ]( event );
+
+				} catch (e) {
+
+					if ( window.console !== undefined && console.error !== undefined ) {
```

Comment:
I think it's a bit overkill. We used to have that in [Three.js](https://github.com/mrdoob/three.js/blob/master/src/Three.js) itself because IE8.

I would let the user do the patching on their side if needed and keep the code minimal.